docs/topics/dokka-gradle-configuration-options.md

```diff
@@ -0,0 +1,871 @@
+        <p>Default: <code>true</code></p>
+    </def>
+    <def title="jdkVersion">
+        <p>The JDK version to use when generating external documentation links for Java types.</p>
+        <p>
+            For example, if you use <code>java.util.UUID</code> in some public declaration signature,
+            and this option is set to <code>8</code>, Dokka generates an external documentation link
+            to <a href="https://docs.oracle.com/javase/8/docs/api/java/util/UUID.html">JDK 8 Javadocs</a> for it.
+        </p>
+        <p>Default: JDK 8</p>
+    </def>
+    <def title="languageVersion">
+        <p>
+            <a href="https://kotlinlang.org/docs/compatibility-modes.html">The Kotlin language version</a>
+            used for setting up analysis and <a href="https://kotlinlang.org/docs/kotlin-doc.html#sample-identifier">@sample</a>
+            environment.
+        </p>
+        <p>By default, the latest language version available to Dokka's embedded compiler is used.</p>
+    </def>
+    <def title="apiVersion">
+        <p>
+            <a href="https://kotlinlang.org/docs/compatibility-modes.html">The Kotlin API version</a>
+            used for setting up analysis and <a href="https://kotlinlang.org/docs/kotlin-doc.html#sample-identifier">@sample</a>
+            environment.
+        </p>
+        <p>By default, it is deduced from <code>languageVersion</code>.</p>
+    </def>
+    <def title="sourceRoots">
+        <p>
+            The source code roots to be analyzed and documented.
+            Acceptable inputs are directories and individual <code>.kt</code> / <code>.java</code> files.
+        </p>
+        <p>By default, source roots are deduced from information provided by the Kotlin Gradle plugin.</p>
+    </def>
+    <def title="classpath">
+        <p>The classpath for analysis and interactive samples.</p>
+        <p>This is useful if some types that come from dependencies are not resolved/picked up automatically.</p>
+        <p>This option accepts both <code>.jar</code> and <code>.klib</code> files.</p>
+        <p>By default, the classpath is deduced from information provided by the Kotlin Gradle plugin.</p>
+    </def>
+    <def title="samples">
+        <p>
+            A list of directories or files that contain sample functions which are referenced via the
+            <a href="https://kotlinlang.org/docs/kotlin-doc.html#sample-identifier">@sample</a> KDoc tag.
+        </p>
+    </def>
+</deflist>
+
+### Source link configuration
+
+Configure source links
+to allow navigation from the generated documentation to the corresponding source code in a remote repository.
+Use the `dokkaSourceSets.main{}` block for this configuration.
+
+The `sourceLinks` configuration block allows you to add a `source` link to each signature
+that leads to the `remoteUrl` with a specific line number.
+The line number is configurable by setting `remoteLineSuffix`.
+
+This helps readers to find the source code for each declaration.
```

Comment:
This sentence can be embedded in the first paragraph: "Configure source links to help readers find the source for each declaration in a remote repository." which is more to the point IMO.